detekt.yml

```diff
@@ -0,0 +1,15 @@
+comments:
+  UndocumentedPublicClass:
+    active: true
+    includes: "**/commonMain/**"
+  UndocumentedPublicProperty:
+    active: true
+    includes: "**/commonMain/**"
+style:
+  UnnecessaryAbstractClass:
+    active: false
```

Comment:
I've disabled this since it behaves weird with Kotlin Multiplatform and doesn't detect expect/actual usages